beast/observationmodel/noisemodel/generic_noisemodel.py

```diff
@@ -10,11 +10,13 @@
 can be achieved using the trunchen method.  The trunchen method
 requires significantly more complicated ASTs and many more of them.
 """
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
 
 import numpy as np
 import tables
 
-import toothpick
+from . import toothpick
```

Comment:
Why do we have a separe toothpick if in "generic_noisemodel" we make more toothpick functions?